src/command/encrypt.iced

```diff
@@ -89,16 +90,22 @@ exports.Command = class Command extends Base
     # We tentatively resolve usernames of the form twitter://foo to
     # foo_keybase, but we still need to assert it's the right person
     # later on.
-    await User.resolve_user_name { username : @argv.them[0] }, esc defer them_un, assertions
+    target_unames = @argv.them[0].split ","
```

Comment:
If we take multiple positional args, we can get rid of the split here and just have

    target_unames = @argv.them